db/migrate/20210629103240_create_participant_profiles.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+class CreateParticipantProfiles < ActiveRecord::Migration[6.1]
+  def change
+    create_table :participant_profiles do |t|
+      t.string :type, null: false
+
+      t.references :user, foreign_key: true, index: true, null: false
+      t.references :school, foreign_key: true, index: true, null: false
+      t.references :core_induction_programme, foreign_key: true, index: true, null: true
+      t.references :cohort, foreign_key: true, index: true, null: false
+
+      # No foregin key yet - not all records will be present
+      t.references :mentor_profile, foreign_key: false, index: true, null: true
```

Comment:
Why won't all the records be present? Cause our migration will take a while?
If we ensured we migrate the mentor profiles first, they should all be present, right?

Reply:
There's no migration to populate the records in this PR - we'll be just creating new records after this is merged. Merging this first means that we don't need any downtime when migrating the existing records (as the new records created when migration is running won't be lost).